scripts/backup_outputs.py

```
@@ -0,0 +1,41 @@
+
+# Get the current working directory
+current_dir = os.getcwd()
+
+# Add it to sys.path
+if current_dir not in sys.path:
+    sys.path.append(current_dir)
+
+current_dir = Path(os.getcwd())
+
+if os.path.basename(current_dir.resolve()) != "clp-bench":
+    raise Exception("Must be run in clp-bench directory, if it was renamed, comment this line out")
```

Ruff (0.12.2)

Create your own exception

(TRY002)